server/Makefile

```diff
@@ -51,13 +58,25 @@ ifeq ($(BUILD_NUMBER),dev)
 	GOTAGS += "testlicensekey"
 endif
 
+# mmctl
+MM_SERVER_PATH ?= $(PWD)
```

Comment:
Not a blocker but what do you think if we there is a need of keeping this?

Reply:
Let me do a quick test to try and remove it. We definitely need to set it on the test targets (on server for example is done in the `scripts/test.sh` file, for `mmctl` we do it directly) but maybe we can simply use `$(PWD)` directly